src/core_write.cpp

```diff
@@ -162,6 +163,7 @@ void ScriptPubKeyToUniv(const CScript& scriptPubKey,
 
     if (!ExtractDestinations(scriptPubKey, type, addresses, nRequired) || type == TX_PUBKEY) {
         out.pushKV("type", GetTxnOutputType(type));
+        out.pushKV("desc", InferDescriptor(scriptPubKey, DUMMY_SIGNING_PROVIDER)->ToString());
```

Comment:
`desc` is always present, could just at the beginning? Like before `asm`.

Reply:
done